### ISSUE TITLE
Using Launch Clamps produces a lot of exception

### ISSUE THREAD

**Issue by BasConijn** [ClawKSP/KSP-Stock-Bug-Fix-Modules#7]
I am getting this exception when adding Launch Clamps in the VAB. 
They don't even have to be attached to the spacecraft.
This only happens in the editor, not in flight.

Exception:

```
[EXC 20:57:26.961] NullReferenceException: Object reference not set to an instance of an object
ClawKSP.LCFix.FixedUpdate ()
[EXC 20:57:26.962] NullReferenceException: Object reference not set to an instance of an object
ClawKSP.LCFix.FixedUpdate ()
```

I original thought this was caused by the FASA Lauch Clamps, but it also happens with the normal clamps.

I have version 1.0.4c1 installed with CKAN.

**Comment by ClawKSP** [comment]
Sorry about that. It should be fixed now in version 1.0.4c.2

**Comment by BasConijn** [comment]
Thanks for the update!
